flake.nix

```diff
@@ -10,7 +12,11 @@
     prometheus-haskell.url = "github:juspay/prometheus-haskell/more-proc-metrics";
     prometheus-haskell.inputs.haskell-flake.follows = "common/haskell-flake";
 
-    euler-hs.url = "github:juspay/euler-hs/ag/open-source";
+    euler-hs = {
+      url = "github:juspay/euler-hs/ag/open-source";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.haskell-flake.follows = "haskell-flake";
```

Comment:
It is unfortunate that we can't use `"common/haskell-flake"` directly here. Is there an upstream issue for it?